homeassistant/components/demo/lock.py

```diff
@@ -75,6 +77,16 @@ def is_locked(self) -> bool:
         """Return true if lock is locked."""
         return self._state == STATE_LOCKED
 
+    @property
+    def is_open(self) -> bool:
+        """Return true if lock is open."""
+        return self._state == STATE_OPEN
+
+    @property
+    def is_opening(self) -> bool:
+        """Return true if lock is opening."""
+        return self._state == STATE_OPENING
```

Comment:
May be add a test to ensure code test coverage.

Reply:
Was a mistake in lock state which made this but it's now also tested properly 👍